### ISSUE TITLE
Update Regression Test documentation 

### ISSUE THREAD

**Issue by a11ydoer** [w3c/aria-practices#1400]
According to [APG meeting today](https://www.w3.org/2020/05/05-aria-apg-minutes.html) and the request from @jongund , APG group needs to update [Regression test documentation](https://github.com/w3c/aria-practices/wiki/Regression-Tests-for-APG-Example-Pages) and [Introduction to APG Regression Tests](https://github.com/w3c/aria-practices/wiki/Introduction-to-APG-Regression-Tests)

**Comment by a11ydoer** [comment]
Please refer @jongund's [report](https://github.com/w3c/aria-practices/pull/1017#issuecomment-624331225).

**Comment by a11ydoer** [comment]
@zcorpan do you have any thing I can help with this issue?

**Comment by a11ydoer** [comment]
@smhigley You mentioned t.file problem. Can you share the detailed info?

**Comment by jongund** [comment]
@zcorpan 
I think there needs to be some information on what systems the regression testing is actually verified to run correctly.   I wasted a lot of time trying to figure out regression test errors when developing on my Windows 10 computer, I ran the same code on my Mac and there were no errors.  So there should be be some information on what systems the regression tests should work, especially for getting PR accepted.
